vncviewer/win32TouchHandler.cxx

```diff
@@ -0,0 +1,251 @@
+/* Copyright 2020 Samuel Mannehed for Cendio AB
+ *
+ * This is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307,
+ * USA.
+ */
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "i18n.h"
+#include "win32TouchHandler.h"
+
+#define XK_MISCELLANY
+#include <rfb/keysymdef.h>
+#include <rfb/LogWriter.h>
+static rfb::LogWriter vlog("win32TouchHandler");
+
+const DWORD MOUSEMOVE_FLAGS = MOUSEEVENTF_MOVE | MOUSEEVENTF_ABSOLUTE |
+                              MOUSEEVENTF_VIRTUALDESK;
+
+static void dbgPrint(DWORD id, GHEvent ghev) {
```

Comment:
We generally put `{` on a new line for top level stuff.

Reply:
Got it, fixed in c0ab26cf8654f8e7ae5bf81dcf0623bd2a0bed94